modules/features/discover/src/main/res/layout/row_podcast.xml

```diff
@@ -21,7 +21,7 @@
 
     <LinearLayout
         android:layout_width="0dp"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
         android:layout_weight="1"
```

Comment:
It might be nice to vertically center the artwork as the text grows.

```
    <androidx.cardview.widget.CardView
        android:layout_width="56dp"
        android:layout_height="56dp"
        android:elevation="2dp"
        app:cardCornerRadius="4dp"
        android:layout_marginRight="8dp"
+++        android:layout_gravity="center_vertical">
```

| Before | After |
| --- | --- |
| ![Screenshot_20250129_162720](https://github.com/user-attachments/assets/76c4c2d7-d94d-4ec3-a57d-c173bc549b52) | ![Screenshot_20250129_162805](https://github.com/user-attachments/assets/f1fbcd0c-5fc5-43d9-b999-66e7bd153f30) |

Reply:
Nice catch! I've updated the PR